Editor/CoreAPI/Models/CreateFleetResponse.cs

```diff
@@ -1,23 +1,9 @@
-﻿using System;
```

Comment:
As mentioned above, if it is possible to return the Response from the API wrapped with a `Response.Ok(...)`, that would help avoid having to introduce new types. The `Response` object accepts a templatized type so i think it should be do-able? 

If this is the case, please discard `CreateFleetResponse`, `DescribeFleetAttributeResponse`, `RegisterComputeResponse`

Reply:
Removed but will be resolved in another PR or after merge.